packages/story-editor/src/app/story/useStoryReducer/reducers/utils.ts

```diff
@@ -155,11 +198,11 @@ export function removeDuplicates(entries = []) {
  * Takes to sets of entries and returns unique entries
  * (keying on id) of right set not present in left set.
  *
- * @param {Array<Entry>} left - base set of entries
- * @param {Array<Entry>} right - new entries
- * @return {Array<Entry>} - right exclusion of sets set
+ * @param left - base set of entries
+ * @param right - new entries
+ * @return - right exclusion of sets set
  */
-export function exclusion(left = [], right = []) {
+export function exclusion(left: Entry[] = [], right: Entry[] = []) {
```

Comment:
If we type this (and `removeDuplicates` above) with `<T extends Entry>`, it's cleaner to use and you can avoid the casts like `exclusion(...) as Element[]` that I see in many places.